editions/tw5.com/tiddlers/howtos/KeyboardShortcuts.tid

```diff
@@ -1,18 +1,62 @@
 created: 20140526204527418
 modified: 20160610082117953
-tags: Features [[Working with TiddlyWiki]]
+tags: Features [[Working with TiddlyWiki]] Keyboard
```

Comment:
Is there a reason for the "Keyboard" tag?

Reply:
not really, no